src/hotspot/share/compiler/methodLiveness.cpp

```diff
@@ -222,6 +222,9 @@ void MethodLiveness::init_basic_blocks() {
           dest = _block_map->at(bytes.get_dest());
           assert(dest != nullptr, "branch destination must start a block.");
           dest->add_normal_predecessor(current_block);
+
+          if (bci + Bytecodes::length_for(code) >= method_len) break;
+
```

Comment:
Please, use code style which use `{}` and put body on separate line as you did in `c1_GraphBuilder.cpp`
You will not need empty lines around it then.